src/pages/ask.tsx

```diff
@@ -1,10 +1,11 @@
-import SortingHat from 'components/survey/sorting-hat'
+import Survey from 'components/survey/survey'
 import * as React from 'react'
+import {sortingHatInitialState} from '../components/survey/survey-reducer'
 
 const Ask: React.FunctionComponent = () => {
   return (
     <div>
-      <SortingHat></SortingHat>
+      <Survey initialSurveyState={sortingHatInitialState} />
```

Comment:
The purpose of this PR is basically here where I'm demonstrating that this is a generic survey and NOT the `sorting hat` so we can pass in ANY `SurveyState` and it will build a survey based on the data

Reply:
survey: used to ask any sequence of questions in a branching fashion
sorting hat: specific implementation of a survey defined by its unique SurveyState
we can implement a "CTA survey" to personalize the active CTA, just as we implemented the sorting hat to reach a final message/state.